internal/proto/v2/common/common.proto

```diff
@@ -53,3 +53,25 @@ enum AlpnProtocol {
   ALPN_PROTOCOL_HTTP2 = 2;
   ALPN_PROTOCOL_HTTP1_1 = 3;
 }
+
+message Identity {
```

Comment:
It looks like we have duplicate copies of the Identity message - is that intentional?

Reply:
I wonder if it makes sense to `move` the Identity here, instead of creating a new copy of definition?

Reply:
Yes -- duplicating ```Identity``` messsage was intentional in ```common.proto``` and ```v2/common.proto```, as discussed offline a few month's ago:
 "we could also add the Identity message to the V2 files...combining the two common.proto files into one is probably infeasible, I think, because the enums are incompatible". This change aligns the v2 protos with the copy in grpc-proto being used by the S2A Java client: https://github.com/grpc/grpc-proto/pull/149

Re: I wonder if it makes sense to move the Identity here, instead of creating a new copy of definition?
> I think this would be possible. We could create ```identity.proto``` which just contains the ```Identity``` message, and this could be used by both v1 and v2. IIRC, the goal as to not share protos between v1 and v2, but if we are ok with ```identity.proto``` being shared between the two, I don't see a problem.

Reply:
Thanks! I had forgotten the details of this discussion. :)